pkg/operator/starter.go

```diff
@@ -260,6 +261,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		masterNodeInformer,
 		kubeClient,
 		envVarController,
+		backuphelpers.NewBackupConfig(v1alpha1.EtcdBackupSpec{}),
```

Comment:
use the disabled version from above, please

Reply:
done